src/NativeUi.elm

```diff
@@ -63,12 +77,34 @@ property =
     Native.NativeUi.property
 
 
-{-| -}
-renderProperty : String -> Decoder a -> (a -> Node b) -> Property msg
+{-| Returns a property representing a rendering function
+
+This is usually used for properties where the values passed to the rendering
+function where created in Elm. To decode values passed from JavaScript, use
+`renderDecodedProperty`.
+
+-}
+renderProperty : String -> (a -> Node b) -> Property msg
 renderProperty =
     Native.NativeUi.renderProperty
 
 
+{-| Returns a property representing a rendering function
+
+Runs values through the decoder before being passed to the rendering function.
+
+This is usually used for properties where the values passed to the
+rendering function were created in JavaScript, and thus need to be decoded. If
+the values are passed from Elm, you can use `renderProperty` to avoid
+encoding/decoding.
+
+Crashes the program if decoding fails.
+-}
+renderDecodedProperty : String -> Decoder a -> (a -> Node b) -> Property msg
```

Comment:
This could be called `dangerouslyRenderDecodedProperty`, unless we can make it safe.